config/cacert.pem

```diff
@@ -1843,8 +1577,8 @@ A2gAMGUCMGYhDBgmYFo4e1ZC4Kf8NoRRkSAsdk1DPcQdhCPQrNZ8NQbOzWm9kA3bbEhCHQ6qQgIx
 AJw9SDkjOVgaFRJZap7v1VmyHVIsmXHNxynfGyphe3HR3vPA5Q06Sqotp9iGKt0uEA==
 -----END CERTIFICATE-----
 
-NetLock Arany (Class Gold) Főtanúsítvány
-============================================
+NetLock Arany (Class Gold) F?tan?s?tv?ny
```

Comment:
something went wrong with the file encoding it seems?

Reply:
master contains an utf8 file, while the download gives me an ascii file. isnt .pem always ascii?